### ISSUE TITLE
OTMultiClass.nii and Weightmaps for training

### ISSUE THREAD

**Issue by satrajitgithub** [Kamnitsask/deepmedic#141]
Hi,

I looked at the training data and found that it has 3 files other than the T1c, FLAIR and mask:
OTMulticlass
Weightmap0
Weightmap1

I was wondering if these are required for training and if required how do I generate them?

Thanks.

**Comment by Kamnitsask** [comment]
Hello,

OTMulticlass is the ground truth segmentation for training.

Weightmaps are not used. They are for a particular type of sampling for training but it's not used. I just removed them in commit d7a11cd as they were confusing users apparenty.

Thanks!
Kostas

**Comment by satrajitgithub** [comment]
Thank you Kostas!

**Comment by Kamnitsask** [comment]
Negative. See TestConfig.
